packages/salesforcedx-vscode-soql/src/lspClient/orgMetadata.ts

```diff
@@ -79,20 +76,12 @@ export class FileSystemOrgDataSource implements OrgDataSource {
 
     let filePath = path.join(soqlMetadataPath, STANDARDOBJECTS_DIR, sobjectName + '.json');
     try {
-      const fileUri = vscode.Uri.file(filePath);
-      const fileStat = await vscode.workspace.fs.stat(fileUri);
-      if (!fileStat) {
-        filePath = path.join(soqlMetadataPath, CUSTOMOBJECTS_DIR, sobjectName + '.json');
-      }
-    } catch {
+      await stat(filePath);
```

Comment:
is this also an existence check?  

I like try/catch for handling errors, but this seems to be more of a "may or may not be there, fine either way" and I like your `fileOrFolderExists` with if/else for that.

or are we more concerned about `catch` for the "readFile might not work"?

If it's that, maybe just call readFile and let the catch handle it not existing?